scripts-2.0/components/TaskPicker/hook.tsx

```diff
@@ -7,7 +7,7 @@ export interface TaskPickerHook {
 }
 
 export const useTaskPickerHook = () => {
-    const emptyTask = {id: 0, parentId: 0, name: "", children: [], path: [], billable: false};
+    const emptyTask = {id: 0, parentId: 0, name: "", children: [], path: [], billable: false, externalTaskId: null};
```

Comment:
nie wiem czy nie lepiej zamaist nulla to dac po prostu `0` tak jak ma `parentId`, chodzi mi tutaj zeby to wszystko spojne bylo, bo `parentId` jak i `externalTaskId` to pewnie ten sam typ

Reply:
wyjasnione, to string przeca